src/test/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilderTest.java

```diff
@@ -156,4 +157,24 @@ void testBillingModeIsPayPerRequestForDefaultCreateTableRequestsWithPayPerReques
                 .collect(Collectors.toList());
     }
 
+    @Test
+    void testTableBuilderInterface() {
+        // Weak cast only for TableBuilder testing purposes
+        TableBuilder tableBuilder = SharedTableBuilder.builder()
+            .withAmazonDynamoDb(LOCAL_DYNAMO_DB)
+            .withTablePrefix(tablePrefix)
+            .withPrecreateTables(true)
+            .withContext(MT_CONTEXT);
+
+        tableBuilder.withBillingMode(BillingMode.PAY_PER_REQUEST);
+
+        ((SharedTableBuilder) tableBuilder).build();
```

Comment:
what happens to the return value?

Reply:
I fixed this one and added comments explaining 9675c3f4b4cd9f76a191e04349d5b2de589ef966